x-pack/plugins/siem/public/components/timeline/index.tsx

```diff
@@ -111,35 +111,39 @@ type Props = OwnProps & StateReduxProps & DispatchProps;
 
 class StatefulTimelineComponent extends React.Component<Props> {
   public shouldComponentUpdate = ({
+    id,
+    flyoutHeaderHeight,
+    flyoutHeight,
+    activePage,
```

Comment:
nit -> I know that you like stuff in order, so just point it out

Reply:
that's true 😄 !

The props are listed in exactly the same order as they appear in `OwnProps & StateReduxProps`, in an attempt to make it easy to verify that when a prop is added, it's not missed. Would you prefer mixing / re-ordering the two prop types within `shouldComponentUpdate` to achieve "true" alphabetical ordering?

Reply:
Sorry, I did not realize that. make sense!!!